torch/ao/quantization/utils.py

```diff
@@ -372,7 +373,7 @@ def type_before_parametrizations(module):
     else:
         return type(module)
 
-def is_leaf_or_only_parametrized(module):
+def has_no_children_ignoring_parametrizations(module):
```

Comment:
Just a side note. But if you move functions to the parametrize module, then we do enforce the FC/BC rules. In particular, you are not allowed to just rename a public function. It should be deprecated first and then deleted. Details here: https://github.com/pytorch/pytorch/wiki/PyTorch's-Python-Frontend-Backward-and-Forward-Compatibility-Policy

Reply:
Yeah, my git-fu is not as strong as it could be, I meant for that to be changed to be in the base commit which is now rectified. The function didn't exist before this stack so hopefully, after moving the creation of the function into the base commit this satisfies the BC requirements.